shared/devices/index.stories.js

```diff
@@ -27,6 +27,7 @@ const provider = Sb.createPropProviderWithCommon({
     firstItem: deviceID === '1',
     isCurrentDevice: deviceID === '1',
     isRevoked: !['1', '2', '3'].includes(deviceID),
```

Comment:
you likely want to add a story for this so we can see it and ensure it doesn't regress

Reply:
👍